src/qt/guiutil.cpp

```diff
@@ -165,6 +167,13 @@ bool parseBitcoinURI(const QUrl &uri, SendCoinsRecipient *out)
             rv.label = i->second;
             fShouldReturnFalse = false;
         }
+        if (i->first == "IS")
+        {
+            if(i->second.compare(QString("1")) == 0)
+                rv.fUseInstantSend = true;
```

Comment:
Well, we already rely on it here in `i->first == "IS"` etc... Btw I don't have strong objection on the way it's implemented, mostly wondering if there any reason to use another approach :)